website/src/pages/BlogPage/slack-icon.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import useStyles from "./style";
+import { useScript } from '../../hooks/useScript';
+
+export const SlackIcon: React.FC = () => {
+    const classes = useStyles();
+    const addThisScript = useScript('//s7.addthis.com/js/300/addthis_widget.js#pubid=ra-60a793659d5a70ad');
+
+    return (addThisScript === 'ready') ? (
+        <div className={["addthis_inline_share_toolbox", classes.socialIconButton].join(' ')}></div>) : ( <div className={classes.socialIconButton}></div> 
+    )
```

Comment:
```suggestion
    return
        <div className={[ (addThisScript === 'ready') ? "addthis_inline_share_toolbox" : "", classes.socialIconButton].join(' ')}></div></div> 
    )
```

Reply:
If we don't add `<div className={classes.socialIconButton}></div>` as fallback, there will be slight shift once slack icon will appear. In order  to avoid shift, I have added fallback `div`.